dist/tools/testrunner/testrunner.py

```diff
@@ -37,7 +37,10 @@ def run(testfunc, timeout=10, echo=True, traceback=False):
         return 1
     finally:
         print("")
-        os.killpg(os.getpgid(child.pid), signal.SIGKILL)
-        child.close()
+        try:
+            os.killpg(os.getpgid(child.pid), signal.SIGKILL)
+            child.close()
+        except OSError:
+            pass
```

Comment:
I hardly call not closing anything at all "gracefully". `native` might block execution if `child.close()` is not finished properly.